src/cogs/admin.py

```diff
@@ -37,7 +34,7 @@ async def reload(self, ctx):
         """
         ReLoads all cogs.
         """
-        if ctx.message.author.id not in self.file["permitted"]:
+        if ctx.message.author.id not in conf.permitted:
             return await ctx.send("You do not have authorization to use this command")
```

Comment:
I thought we were gonna change this to a decorator?

Reply:
We only discussed doing so for the spam detection specific commands.